fossilize.cpp

```diff
@@ -4363,11 +4576,18 @@ void StateRecorder::Impl::pump_synchronized_recording(StateRecorder *recorder)
 bool StateRecorder::record_sampler(VkSampler sampler, const VkSamplerCreateInfo &create_info, Hash custom_hash)
 {
 	{
-		if (create_info.pNext)
+		// Only custom border color and reduction mode are supported for now.
+		auto *custom_border_color = find_pnext<VkSamplerCustomBorderColorCreateInfoEXT>(
```

Comment:
This is not necessary to check explicitly. We'll just error out in copy_pnext_chain instead.